image/db/rhel/Dockerfile

```diff
@@ -27,19 +27,19 @@ COPY scripts/docker-entrypoint.sh /usr/local/bin/
 COPY --from=extracted_bundle /bundle/postgres.rpm /bundle/postgres-libs.rpm /bundle/postgres-server.rpm /tmp/
 COPY --from=extracted_bundle /bundle/etc/postgresql.conf /bundle/etc/pg_hba.conf /etc/
 
-RUN groupadd -g 70 postgres && \
+RUN microdnf upgrade && \
+    microdnf install \
+        shadow-utils libicu systemd-sysv /tmp/postgres-libs.rpm \
```

Comment:
`ca-certificates` comes pre-installed, so it's not needed here. However, we need `shadow-utils` for `groupadd` and `adduser`